pages/events/index.vue

```diff
@@ -75,6 +75,9 @@
             :to="event.to"
           />
         </div>
+        <div v-else class="event-page__results">
+          <h3>Plan an event in your area today!</h3>
```

Comment:
Non-blocking: knowing this is a placeholder but just-in-case, could you make start the placeholder with something indicating there are no events matching this criterion. A simple `No events: plan an event...` is enough.